nodes/pods/nodes-pods-secrets.adoc

```diff
@@ -27,6 +27,8 @@ include::modules/nodes-pods-secrets-about.adoc[leveloffset=+1]
 
 include::modules/nodes-pods-secrets-creating.adoc[leveloffset=+1]
 
+include::modules/nodes-pods-secrets-creating-web-console-secrets.adoc[leveloffset=+2]
```

Comment:
This section might make more sense to come after all of the other existing CLI procedures. Since you don't clarify in the titles that those are all CLI, after you've said web console for the first one. (And I don't think that's necessary, but reordering should be enough)

So I think it would make sense to show all of those existing ways to create the various types of secrets by using the CLI. And then at the end, it's like oh yeah, here's how to create those secrets through the web console, if you wanted to do that instead.

So I'd consider moving it down to after "include::modules/nodes-pods-secrets-creating-docker.adoc[leveloffset=+2]"